plugin/storage/cassandra/spanstore/writer.go

```diff
@@ -63,8 +63,13 @@ const (
 	defaultNumBuckets = 10
 
 	durationBucketSize = time.Hour
+
+	indexOnly     = storageMode(1)
```

Comment:
* use the `iota` pattern
* I would put indexAndStore as the first item (the default mode)